client/signup/steps/domains/index.jsx

```diff
@@ -1011,9 +1015,9 @@ export class RenderDomainsStep extends Component {
 
 		return (
 			<ProvideExperimentData
-				name="calypso_gf_signup_onboardingpm_domains_hide_free_subdomain"
+				name="calypso_gf_signup_onboardingpm_domains_hide_free_subdomain_v2"
 				options={ {
-					isEligible: includeWordPressDotCom && this.props.flowName === 'onboarding-pm',
+					isEligible: this.props.flowName === 'onboarding-pm',
```

Comment:
`includeWordPressDotCom` is removed here since I want the condition here to be aligned with the `loadExperimentAssignment()` call above.

Reply:
What is `includeWordPressDotCom` used for?

Reply:
It is a flag indicating that the wp.com free subdomain should be included in the search result or not. In our `onboarding-pm` flow, it should be always `true` though.